cartocrow/isoline_simplification/ipe_bezier_wrapper.h

```diff
@@ -0,0 +1,37 @@
+/*
+The CartoCrow library implements algorithmic geo-visualization methods,
+developed at TU Eindhoven.
+Copyright (C) 2024 TU Eindhoven
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3f of the License, or
```

Comment:
I'm wondering where the `3f` is coming from. This should just be version 3.

Reply:
I don't know, it was also in some flow map header so I think I copied it from there. Fixed now.